### ISSUE TITLE
Change Titles on All Pages

### ISSUE THREAD

**Issue by xmido** [a-shaheen/bookish-spoon#2]
Site looks great, ut it looks like pages other than the index.html page have the wrong title.

**Comment by a-shaheen** [comment]
That's right... duh. I forgot to change the other pages. I'll take care of it right away.

**Comment by a-shaheen** [comment]
Thanks for the  tip @xmido the changes have been made.
